src/index.html

```diff
@@ -370,17 +373,21 @@
                 </div>
             </div>
         </div>
-<!--         <div id="maps" class="container tab-pane">
-            <div class="row">
-                <div class="col-12 p-5">
+        <!-- META-DATA TEXT BOX -->
 
-                    <div style="max-width: 1000px">
-                        <div id="mapid"></div>
-                    </div>
-                    <button class="btn btn-success" data-bind="click: renderMap">Redraw Plot</button>
-                </div>
+        <div class="card" style="width: 50rem;">
+            <div class="card-header">
+                <a class="nav-link nav-parent"
+                   data-toggle="collapse"
+                   href="#showHideMeta"
+                   aria-expanded="false"
+                   aria-controls="showHideMeta"
+                   >Show / Hide metadata</a>
```

Comment:
You could switch this text depending on what its current state is - either 'Show metadata' or 'Hide metadata'. Will 'metadata' be meaningful for all users? Would 'description' make more sense?